prow/genjobs/README.md

```diff
@@ -6,16 +6,39 @@
 
 ## Installation
 
+Install using Golang:
+
+```console
+$ GO111MODULE="on" go get -u istio.io/test-infra/prow/genjobs
+```
+
+Install using Docker:
+
 ```console
-$ go get -u istio.io/test-infra/prow/genjobs
+$ docker pull gcr.io/istio-testing/genjobs:latest
+```
+
+Install from source:
+
+```console
+$ git clone --depth=1 https://github.com/istio/test-infra.git
```

Comment:
This will be easier for users to consume if you get rid of the `$`

Reply:
As a user, it clear that this is a terminal command with the `$` prefix?

Reply:
Updated